src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -98,9 +108,13 @@ public void loadExistingCredentials() {
     // load existing user credentials from disk
     try (InputStream inputStream =
         User.class.getClassLoader().getResourceAsStream(CLIENT_SECRET_FILENAME)) {
-      userCredentials =
-          GoogleOauth.getExistingUserCredential(
-              USER_SCOPES, inputStream, Context.getContextDir().toFile());
+      if (useApplicationDefaultCredentials) {
```

Comment:
I think you should move this check outside of the try-with-resources, since we don't need to find & open the file for this option.